package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@neo9/n9-node-log",
-  "version": "2.6.0",
```

Comment:
@victorboissiere  We change the version only on release

Reply:
I guess you will do the next release yourself then, thats fine